### DISCUSSION TITLE
Adding the exit animation for lazer on mobile

### DISCUSSION THREAD

**Discussion by darusei** [ppy/osu#19493] [Ideas] [answered]
I played osu!lazer on PC and for exiting i always (long) pressed `Esc` with the "See You next time". But i mostly play on my Android phone and pressing the "Back" button it just closes the game without the voice. I really like this game and wanted to share the idea of adding the exit voice on mobile devices too.
Greetings from Italy.

**Comment by nagi-desuuu** [comment, marked as answer]
https://github.com/ppy/osu/discussions/16546
